dbcon/joblist/tuple-bps.cpp

```diff
@@ -180,10 +180,8 @@ void TupleBPS::initializeConfigParms()
     fNumThreads = 0;
 
     config::Config* cf = config::Config::makeConfig();
-    string epsf = cf->getConfig("ExtentMap", "ExtentsPerSegmentFile");
 
-    if ( epsf.length() != 0 )
-        fExtentsPerSegFile = cf->uFromText(epsf);
+    fExtentsPerSegFile = 2;
```

Comment:
Could you use a contstant you define previousl, namely DEFAULT_EXTENTS_PER_SEGMENT_FILE ?